http/jsonapi/generator/generate_test.go

```diff
@@ -34,7 +36,15 @@ func TestGenerator(t *testing.T) {
 				t.Fatal(err)
 			}
 
+			if updatedExpectedFiles {
```

Comment:
`updatedExpectedFiles` is always false. This looks like you did this for debugging reasons and forgot to change it back.